tests/callbacks/test_swa.py

```diff
@@ -157,3 +157,30 @@ def test_swa_raises():
         StochasticWeightAveraging(swa_epoch_start=-1, swa_lrs=0.1)
     with pytest.raises(MisconfigurationException, match="positive float or a list of positive float"):
         StochasticWeightAveraging(swa_epoch_start=5, swa_lrs=[0.2, 1])
+
+
+@pytest.mark.parametrize('use_callbacks', [False, True])
+@pytest.mark.parametrize('stochastic_weight_avg', [False, True])
+def test_trainer_and_stochastic_weight_avg(tmpdir, use_callbacks, stochastic_weight_avg):
```

Comment:
Should flip the decorator order here: The decorator that is at the bottom populates the first argument (excluding the tmpdir fixture)

Reply:
good catch !